scoring/templates/scoring/methodology.html

```diff
@@ -24,6 +24,7 @@ <h3>Disclaimer</h3>
                     council pages. Visit the <a href="https://data.climateemergency.uk/">Climate Action Plan Explorer</a> to see if there is a later version of a council’s plan.</span>
             </div>
             <p><strong>To find out about the history of and methodology behind the Scorecards, please go to </strong> <a class="d-inline-block" href="{% url 'about' %}">About us</a></p>
+            <p><a href="https://mysociety.github.io/climate_scorecard_data_2022/">Download the raw dataset in spreadsheet form</a>.</p>
```

Comment:
FWIW when an entire sentence is a link, we include the period _in_ the link, rather than outside it.